docs/users/applicants.md

```diff
@@ -0,0 +1,37 @@
+---
+id: applicants
+title: Applicants
+sidebar_label: Applicants
+---
+
+Pupilfirst allows you to collect applications for your courses, and review them before adding students to a course. This is useful if you want to restrict access to a course to a select group of students, or if you want to collect additional information from students before they join a course.
+
+## Enabling applications
+
+Applicant processing is an _optional_ step, the system will auto onboard the student if it not enabled.
+
+To enable applications for a course, The course should have [public signups enabled](/users/courses#creating-courses) and the [applicant processing](/users/courses#creating-courses) option should be enabled.
+
+## Viewing the list of applicants
+
+To view the list of applicants that are waiting to be processed, click on the _Applicants_ rom the course's sub-menu.
+
+![Applicants page within a course in the school administration interface](../assets/applicants/applicants_page.png)
+
+## Onboarding applicants
+
+Click on the _onBoard as Student_ button to add the applicant as a student in the course. This will open up a form that allows you to enter the student's details, and add them to the course.
```

Comment:
Why is `OnBoard` camelcased in the UI..? That's not how _regular_ English works. 😅 This should be `Onboard`.